examples/colourpicker-histogram.py

```diff
@@ -0,0 +1,48 @@
+from dash import Dash, dcc, html, Input, Output
+import plotly.graph_objects as go
+import plotly.express as px
+import pandas as pd
+import dash_daq as daq
+import plotly.io as pio
+
+df = px.data.gapminder()
+df = df.loc[df['year']==1952]
+
+pio.templates.default = "ggplot2"
+
+app = Dash(__name__)
+
+picker_style = {"display": "inline-block", "margin": 10}
```

Comment:
we can remove this `picker_style` line of code, unless you plan to use it somewhere in the app.